packages/plugin/src/processor.ts

```diff
@@ -1,12 +1,31 @@
 import type { Linter } from 'eslint';
-import { parseCode } from '@graphql-tools/graphql-tag-pluck';
+import { parseCode, GraphQLTagPluckOptions } from '@graphql-tools/graphql-tag-pluck';
+import { loadGraphQLConfig } from './graphql-config';
 
 type Block = Linter.ProcessorFile & {
   lineOffset: number;
   offset: number;
 };
 
-const RELEVANT_KEYWORDS = ['gql', 'graphql', '/* GraphQL */'] as const;
+const defaultGraphqlConfig = loadGraphQLConfig({})?.getDefault();
+const graphqlTagPluckOptions = defaultGraphqlConfig?.extensions?.graphqlTagPluck as GraphQLTagPluckOptions;
+const graphqlTagModuleIdentifiers =
+  graphqlTagPluckOptions?.modules.map(({ identifier }) => identifier).filter(identifier => identifier) ?? [];
+const graphqlTagModuleNames = graphqlTagPluckOptions?.modules.map(({ name }) => name).filter(name => name) ?? [];
+const globalGqlIdentifierNames = graphqlTagPluckOptions?.globalGqlIdentifierName
+  ? Array.isArray(graphqlTagPluckOptions.globalGqlIdentifierName)
+    ? graphqlTagPluckOptions.globalGqlIdentifierName
+    : [graphqlTagPluckOptions.globalGqlIdentifierName]
+  : ['gql', 'graphql'];
+const gqlMagicComment = graphqlTagPluckOptions?.gqlMagicComment ?? '/* GraphQL */';
```

Comment:
Default magic comment `/* GraphQL */` should not contain `/*` and `*/` characters, so queries with extra spaces will be also processed, e.g. 
```
/*   GraphQL */
```
Didn't know it, just discovered it from `graphql-tag-pluck` 😆